src/pages/App.tsx

```diff
@@ -78,13 +89,17 @@ export default function App() {
   const currentPage = getCurrentPageFromLocation(pathname)
   const isDarkMode = useIsDarkMode()
   const [routerPreference] = useRouterPreference()
-  const [scrolledState, setScrolledState] = useState(false)
+  const [scrollY, setScrollY] = useState(0)
+  const scrolledState = scrollY > 0
 
   const routerConfig = useRouterConfig()
 
+  const originCountry = useAppSelector((state: AppState) => state.user.originCountry)
+  const renderUkBannner = Boolean(originCountry) && originCountry === 'GB'
```

Comment:
Does GB work for Scotland, Wales, and Northern Ireland?

Reply:
GB is only for the UK

Reply:
Scotland, Wales, and Northern Ireland are part of the UK but might be under different country codes